src/tasks/mt.py

```diff
@@ -128,7 +125,7 @@ def __init__(self, path, max_seq_len, max_targ_v_size, name, **kw):
                          max_targ_v_size=max_targ_v_size, name=name,
                          **kw)
         self._label_namespace = None
-        self.target_indexer = {"words": SingleIdTokenIndexer("tokens")}
```

Comment:
Again, is this a no-op change?

Reply:
yes (by default that arg is "tokens"), but it's worth being explicit